script.js

```diff
@@ -17,3 +23,28 @@ btnmenu.addEventListener('click', () => {
 btnclose.addEventListener('click', () => {
   item.style = 'display: none;';
 });
+
+const data = [];
+submit.addEventListener('click', () => {
+  if (emailval.value === emailval.value.toLowerCase() && emailval.value !== '') {
+    data[0] = Myname.value;
+    data[1] = emailval.value;
+    data[2] = comments.value;
+    localStorage.setItem('local', JSON.stringify(data));
+  } else {
+    terror.textContent = 'error';
+  }
+});
```

Comment:
- [ ] Please adhere to the project requirement according to which your form should validate the contact form input field, especially email. It seems like your input gives the error on wrong input but after entering the correct information, still `error` is there around the submit button and your form submit button event is even not working. Kindly fix the issue so that your portfolio should look professional and anybody can send you his/her query after visiting your portfolio. Make sure that you are putting your good efforts into this portfolio because it will become your actual portfolio in future to showcase your skills, projects and everything. 

### Video

[Screencast from 23-06-23 02:39:03 PM IST.webm](https://github.com/Wils-FOURR-speed/My-portfolio/assets/66476329/645fc365-611c-4a3c-986e-a17e437e3ca6)